dask/array/rechunk.py

```diff
@@ -195,8 +195,9 @@ def blockshape_dict_to_tuple(old_chunks, d):
     shape = tuple(map(sum, old_chunks))
     new_chunks = list(old_chunks)
     for k, v in d.items():
-        div = shape[k] // v
-        mod = shape[k] % v
+        if v == -1:
```

Comment:
Are there other values that we want to use here, like `None`?  Or does that mean don't change anything?  Also, do you have the time to add this behavior to the docstring?

Reply:
Updated the docstrings.

We didn't discuss any alternatives when I added this back in https://github.com/dask/dask/issues/2689, but in https://github.com/pydata/xarray/issues/2103 @crusaderky suggests supporting `np.inf`. That seems pretty reasonable to me, too.